UPGRADE.md

```diff
@@ -1,15 +1,40 @@
-# Upgrade
+Upgrade
+=======
+
+dev-master
+----------
+
+The HTTP cache integration has been refactored. The following configuration
```

Comment:
ok. but I think we *have* to structure it like this, upgrade instructions cannot be implemented as a list.